Deliverable 1/API Examples/sendRankingInteractions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Eval_Request": {
+    "query": "How do I parse a JSON file",
```

Comment:
Sending directly to ranking is problematic, since they won't get the raw query, just the processed one, but I don't think the right move is to send it to query either.

Best solution is to probably convince querying to tell UI UX the raw query along with the query results

Reply:
That sounds good